sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -272,6 +273,12 @@ export function setClientRequestIdPolicy(requestIdHeaderName?: string): Pipeline
 // @public
 export const setClientRequestIdPolicyName = "setClientRequestIdPolicy";
 
+// @public
+export interface SetClientRequestIdPolicyOptions {
+    disable?: boolean;
```

Comment:
so for `core-rest-pipeline` I did a bunch of work to eliminate `disable` style options, since the pipeline is directly configurable. Can we not simply remove the policy on the pipeline when doing the tests instead of setting this config option?

Reply:
I think this makes sense, we could do this in the generator side
![image](https://user-images.githubusercontent.com/21109913/114480113-b9e3cf80-9bb6-11eb-8b70-6531ee9688b6.png)